baseapp/pom.xml

```diff
@@ -12,16 +12,37 @@
     <artifactId>baseapp</artifactId>
     <packaging>jar</packaging>
 
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <dropwizard.version>3.0.1</dropwizard.version>
+        <guice.version>5.1.0</guice.version>
+        <reflections.version>0.9.10</reflections.version>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>io.dropwizard</groupId>
+                <artifactId>dropwizard-dependencies</artifactId>
+                <version>3.0.1</version>
```

Comment:
Since you've added a `dropwizard.version` you could use it here.

Reply:
yes. using it now. I should have stashed the previous revisions.